index.html

```diff
@@ -0,0 +1,231 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,600;0,700;1,400&display=swap" rel="stylesheet">
+  <link href="./styles/index.css" rel="stylesheet">
+</head>
+<body class="main">
+<div class="page1">
```

Comment:
Please try to use descriptive and meaningful names (e.g page1 does not describe which block or part on your page)